source/API/CMakeLists.txt

```diff
@@ -170,7 +170,16 @@ if(LLDB_BUILD_FRAMEWORK)
 endif()
 
 set(lib_dir "${CMAKE_BINARY_DIR}/${CMAKE_CFG_INTDIR}/lib${LLVM_LIBDIR_SUFFIX}")
-set(CLANG_RESOURCE_PATH "${LLDB_PATH_TO_SWIFT_BUILD}/lib${LLVM_LIBDIR_SUFFIX}/swift/clang")
+
+if(EXISTS ${LLDB_PATH_TO_SWIFT_BUILD}/lib${LLVM_LIBDIR_SUFFIX}/swift/clang)
```

Comment:
Did we double-check that we can do these checks at CMake configuration time? Was the directory created already or does this happen at build-time?

Reply:
This happens at configure time, however, the Swift dependency means that Swift must built built prior to the build of lldb.  Just to be extra sure, I'll do a clean build test here.